ambry-router/src/main/java/com/github/ambry/router/CompositeBlobOperationHelper.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2021 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.router;
+
+import com.github.ambry.protocol.GetOption;
+import java.util.List;
+import java.util.function.Consumer;
+
+
+/**
+ * A helper class that carries information for ttl update and undelete operation on a potential composite blob.
+ */
+public class CompositeBlobOperationHelper {
```

Comment:
Now this is an independent class, can we make it a pojo and encapsulate these fields?

Reply:
done